PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,27 @@
-## Please provide the following details to expedite review (and delete this heading)
-
-*Replace with a description about why this change is needed, along with a description of what changed.*
+*Why this change is needed and what it does.*
 
 ## Checklist
 
- - [ ] Requires a [pylibjuju](https://github.com/juju/python-libjuju) change
- - [ ] Added [integration tests](https://github.com/juju/juju/tree/develop/tests) for the PR
- - [ ] Added or updated [doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) related to packages changed
- - [ ] Comments answer the question of why design decisions were made
+*Strikethrough if not applicable, e.g., `~Go unit tests, with comments saying what you're testing~`.*
```

Comment:
Is the "e.g." needed? I think "Strikethrough if not applicable" is plenty clear.

Reply:
Thanks, updated.